### PR TITLE
ProductController created

### PR DESCRIPTION
**Pull request by Gwynnbleidd11**
ProductController was created along with basic domain classes. I had to comment `application.properties` in order to test if application runs fine. Forgot to un-comment it before push, not sure how to change it now. Please reject if needed and I will correct.

### CODE REVIEW

**Comment by piotr-grosicki** [comment]
Just commit and push next changes to the same branch :)

**Comment by Gwynnbleidd11** [comment]
Next commit done. Application.properties file corrected and UserController created. Please review

File: src/main/java/com/kodilla/ecommercee/product/controller/ProductController.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        return listOfProducts;
+    }
+
+    @GetMapping(value = "{productId}")
+    public ProductDto getProduct(@PathVariable final Long productId) {
+        return new ProductDto(1L, "Koc", "Ciepły koc", 9.99);
+    }
+
+    @PostMapping
+    public void createProduct(final ProductDto productDto) {
```
Comment (piotr-grosicki) [inline src/main/java/com/kodilla/ecommercee/product/controller/ProductController.java:28]: missing RequestBody

File: src/main/java/com/kodilla/ecommercee/product/controller/ProductController.java [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        System.out.println("Creating product");
+    }
+
+    @PutMapping
+    public ProductDto updateProduct(final ProductDto productDto) {
```
Comment (piotr-grosicki) [inline src/main/java/com/kodilla/ecommercee/product/controller/ProductController.java:33]: missing RequestBody

File: src/main/java/com/kodilla/ecommercee/product/controller/ProductController.java [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+        return new ProductDto(2L, "Zaktualizowany koc", "Cieplejszy koc", 12.99);
+    }
+
+    @DeleteMapping
+    public void deleteProduct(final Long productId) {
```
Comment (piotr-grosicki) [inline src/main/java/com/kodilla/ecommercee/product/controller/ProductController.java:38]: missing RequestParam / PathVariable

File: src/main/java/com/kodilla/ecommercee/product/domain/Product.java [hunk 4]
```diff
@@ -0,0 +1,16 @@
+package com.kodilla.ecommercee.product.domain;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@Getter
+@AllArgsConstructor
+@NoArgsConstructor
+public class Product {
```
Comment (piotr-grosicki) [inline src/main/java/com/kodilla/ecommercee/product/domain/Product.java:10]: entity is not needed right now

File: src/main/java/com/kodilla/ecommercee/user/controller/UserController.java [hunk 5]
```diff
@@ -0,0 +1,25 @@
+package com.kodilla.ecommercee.user.controller;
+
+import com.kodilla.ecommercee.user.domain.UserDto;
+import com.kodilla.ecommercee.user.domain.UserUpdateStatusDto;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("/users")
+public class UserController {
```
Comment (piotr-grosicki) [inline src/main/java/com/kodilla/ecommercee/user/controller/UserController.java:9]: why user controller is created on this branch?

**Review by piotr-grosicki — CHANGES_REQUESTED** [review]
remove everything related to User

**Comment by Gwynnbleidd11** [comment]
ProductController corrected, everything related to user removed from this branch. I added User to this branch, as this is how I understood your comment "Just commit and push next changes to the same branch :)". I wasn't expecting any changes to ProductController branch at this stage.

**Pull request merged**